src/psyclone/tests/psyclone_test_utils.py

```diff
@@ -163,6 +180,24 @@ def get_infrastructure_flags(self):
         '''
         return []
 
+    @staticmethod
+    def skip_if_compilation_disabled():
+        '''This function is used in all tests that should only run
+        if compilation is enabled. It calls pytest.skip if compilation
+        is not enabled.'''
+        if not Compile.TEST_COMPILE:
+            return pytest.skip("Need --compile option to run")
+
+    @staticmethod
+    # pylint: disable=invalid-name
```

Comment:
pylint also says that this suppression is "useless" :-)

Reply:
That seems to depend on the pylint version - my previous version was ok with that :)

After the update I now have different pylints for python2 and 3 (1.9.4 and 2.3.1 resp). 

Python3's pylint seems to be ok with the name, but then complains for me about "Class 'Compile' inherits from object, can be safely removed from bases in python3", while python2 is not ok with the name ("doesn't conform to snake_case naming style" - which seems to indicate that the name is too long), but happy with inheritance from object.

Since you don't mention the inheritance from 'object', I've left this unchanged, and hope that you now get a clean pylint output. But it might be worth that we all agree on a certain pylint version, and update it at the same time?

Reply:
I'm not too worried about the details - e.g. the names thing. The inheritance from 'object' is, I think, a Python 2/3 thing and we will have to live with that until we can drop Python 2. In fact we could probably suppress that warning altogether but we can ignore it for now.